Formula/glab.rb

```diff
@@ -18,12 +19,9 @@ class Glab < Formula
   def install
     system "make", "GLAB_VERSION=#{version}"
     bin.install "bin/glab"
-    output = Utils.safe_popen_read({ "SHELL" => "bash" }, bin/"glab", "completion", "bash")
-    (bash_completion/"glab").write output
-    output = Utils.safe_popen_read({ "SHELL" => "zsh" }, bin/"glab", "completion", "zsh")
-    (zsh_completion/"_glab").write output
-    output = Utils.safe_popen_read({ "SHELL" => "fish" }, bin/"glab", "completion", "fish")
-    (fish_completion/"glab.fish").write output
+    (bash_completion/"glab").write `#{bin}/glab completion --shell=bash`
+    (zsh_completion/"_glab").write `#{bin}/glab completion --shell=zsh`
+    (fish_completion/"glab.fish").write `#{bin}/glab completion --shell=fish`
```

Comment:
```suggestion
    (bash_completion/"glab").write Utils.safe_popen_read(bin/"glab", "completion", "--shell=bash")
    (zsh_completion/"_glab").write Utils.safe_popen_read(bin/"glab", "completion", "--shell=zsh")
    (fish_completion/"glab.fish").write Utils.safe_popen_read(bin/"glab", "completion", "--shell=fish")
```

Yeah, it's totally possible that it never worked.

Almost there. I'd like to keep using `Utils.safe_popen_read` rather than the backticks. That way, if the command changes in the future, an error will be thrown letting us know.

Reply:
Makes sense to me!

For (my) future reference: should I _always_ prefer `Utils.safe_popen_read` over backticks when writing new code in Homebrew? And, if so, is there an official Homebrew backlog of desired but not yet implemented audit checks that we could add that rule to?

Reply:
(unresolving the conversation solely because of my pending question)